packages/go/tsconfig.json

```
@@ -11,8 +11,11 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitThis": false,
+    "outDir": "./dist",
     "types": ["node", "jest"],
     "strict": true,
     "target": "ES2021"
-  }
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules"]
```

`node_modules` is excluded by default

```suggestion
  "include": ["src/**/*"]
```